lib/galaxy/dependencies/dev-requirements.txt

```diff
@@ -157,20 +158,18 @@ roc-validator==0.4.2 ; python_full_version < '3.9.20'
 roc-validator==0.4.6 ; python_full_version >= '3.9.20' and python_full_version < '3.10'
 roc-validator==0.7.3 ; python_full_version >= '3.10'
 roman-numerals-py==3.1.0 ; python_full_version >= '3.11'
-rpds-py==0.27.0
+rpds-py==0.27.1
 ruamel-yaml==0.18.15
 ruamel-yaml-clib==0.2.12 ; python_full_version < '3.14' and platform_python_implementation == 'CPython'
-rucio-clients==37.7.1
+rucio-clients==38.0.0
 s3transfer==0.13.1
 schema-salad==8.9.20250723145140
 scipy==1.13.1 ; python_full_version < '3.10'
 scipy==1.15.3 ; python_full_version == '3.10.*'
 scipy==1.16.1 ; python_full_version >= '3.11'
 secretstorage==3.3.3 ; platform_machine != 'ppc64le' and platform_machine != 's390x' and sys_platform == 'linux'
-selenium==4.32.0 ; python_full_version < '3.10'
-selenium==4.35.0 ; python_full_version >= '3.10'
+selenium==4.32.0
```

Comment:
Need to investigate what caused this change.

Reply:
OK, this was caused by the update of typing-extensions to 4.15.0 because selenium 4.35.0 depends on `typing-extensions~=4.14.0` (while selenium 4.32.0 depends on `typing-extensions~=4.9`, i.e `>=4.9,<5`)
Fixed by https://github.com/SeleniumHQ/selenium/pull/16262 .